modules/TabStack.js

```diff
@@ -113,26 +113,38 @@ class TabStack extends Component<DefaultProps, OwnProps, State> {
   onRequestChangeTab = (index: number): void => {
     const entries = this.state.history[index]
     if (index !== this.state.navigationState.index) {
```

Comment:
Is that really necessary ^^'?… I could if you insist, but that would be to cumbersome imho (why not just change the PR title haha?)